content/en/integrations/guide/reference-tables.md

```diff
@@ -88,6 +88,29 @@ Click **New Reference Table +**, then add a name, select AWS S3, fill out all fi
 
 [1]: https://app.datadoghq.com/account/settings#integrations/amazon-web-services
 [2]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#installation
+{{% /tab %}}
+
+{{% tab "Azure storage" %}}
+
+1. If you haven't already, set up the [Azure integration][1] within the subscription that holds the storage account from which you want to import your Reference Table. This involves [creating an app registration that Datadog can use][2] to integrate with.
+2. In the Azure Portal, select the storage account that stores your Reference Table files.
+3. Within your storage account, navigate to **Access Control (IAM)** and select **Add** > **Add Role Assignment**.
+4. Input and select the **Storage Blob Data Reader** Role. The [Storage Blob Data Reader role][3] allows Datadog to read and list storage containers and blobs.
+5. In the **Members** tab, click **+ Select members**. Select the app registration you created in Step 1.
+   
+   {{< img src="integrations/guide/reference-tables/add_members.png" alt="The Members section in the Azure Portal where a member is selected and data filled in for the Name, Object ID, and Type" style="width:85%;">}}
+
+After reviewing and assigning the role, you can import into Reference Tables from Azure. It may take a few minutes for your Azure configuration to update in Datadog.
+
+{{< img src="integrations/guide/reference-tables/azure_storage.png" alt="An Azure Storage tile in the Upload or import data section of a new reference table workflow" style="width:80%;">}}
+
+For more information, see the [Azure integration documentation][4]. 
```

Comment:
```suggestion
For more information, see the [Azure integration documentation][4]. 

**Note**: The upload from cloud object storage supports files up to 200MB.

```

Reply:
Looks good. Generalizing the comment about the 200MB limit to all cloud storage types makes sense.